iiop-openjdk/src/main/resources/schema/wildfly-iiop-openjdk_3_0.xsd

```diff
@@ -137,29 +145,34 @@
+                * server-ssl-context: the dynamic name of the capability that provides the SSL context that will be used
+                        to create server side SSL sockets. If the defined, the client-ssl-context attribute must also be
+                        defined and the security-domain attribute must not be set.
+                * client-ssl-context: the dynamic name of the capability that provides the SSL context that will be used
+                        to create client side SSL sockets. If the defined, the server-ssl-context attribute must also be
+                        defined and the security-domain attribute must not be set.
                 * add-component-via-interceptor: indicates whether SSL components should be added by an IOR interceptor
-                                                 (true) or not (false).
-                * client-supports: value that indicates the client SSL supported parameters (None, ServerAuth, ClientAuth.
-                                   MutualAuth)
-                * client-requires: value that indicates the client SSL required parameters (None, ServerAuth, ClientAuth.
-                                   MutualAuth)
-                * server-supports: value that indicates the server SSL supported parameters (None, ServerAuth, ClientAuth.
-                                   MutualAuth)
-                * server-requires: value that indicates the server SSL required parameters (None, ServerAuth, ClientAuth.
-                                   MutualAuth)
+                        (true) or not (false).
+                * server-requires-ssl: indicates wheter IIOP connections to the server require SSL
+                * client-requires-ssl: indicates wheter IIOP connections from the server require SSL
                         ]]>
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="support-ssl" type="trueFalseType" use="optional" default="false"/>
         <xs:attribute name="security-domain" type="xs:string" use="optional"/>
+        <xs:attribute name="server-ssl-context" type="xs:string" use="optional"/>
+        <xs:attribute name="client-ssl-context" type="xs:string" use="optional"/>
         <xs:attribute name="add-component-via-interceptor" type="trueFalseType" use="optional" default="true"/>
-        <xs:attribute name="client-supports" type="sslConfigType" use="optional" default="MutualAuth"/>
-        <xs:attribute name="client-requires" type="sslConfigType" use="optional" default="None"/>
-        <xs:attribute name="server-supports" type="sslConfigType" use="optional" default="MutualAuth"/>
-        <xs:attribute name="server-requires" type="sslConfigType" use="optional" default="None"/>
+        <xs:attribute name="server-requires-ssl" type="xs:boolean" use="optional" default="false"/>
+        <xs:attribute name="client-requires-ssl" type="xs:boolean" use="optional" default="false"/>
```

Comment:
Also (I can't comment on the line since it was not changed, I think the version attribute of the root xs:schema element should be version=3.0 rather than version=1.0?